src/framework/asset/asset-registry.js

```diff
@@ -362,16 +384,24 @@ class AssetRegistry extends EventHandler {
      *     app.assets.load(assetToLoad);
      * });
      */
-    load(asset) {
+    load(asset, options) {
```

Comment:
asset already has an options field. why add another one here?

Reply:
Not sure I understand the question. These properties are required to pass situational flags based on where `load` is called from.